viewing_party/party.py

```diff
@@ -1,23 +1,178 @@
+from collections import Counter
+
+
 # ------------- WAVE 1 --------------------
 
+
+# creates a movie 
+# returns dictionary
+
 def create_movie(title, genre, rating):
-    pass
+
+    if title == None or genre == None  or rating == None:
+        movie_dict = None
+    else:
+        movie_dict = {"title":title,"genre":genre,"rating":rating}

```

Comment:
This is a matter of personal preference and what reads easiest for you, but since the variable new_movie is only used in the return statement, we could factor it out by directly returning the new dictionary. Something like this:

`return { 
        "title" : movie_title, 
        "genre": genre, 
        "rating" : rating
 }`